src/registry_schemas/schemas/dissolution.json

```diff
@@ -30,6 +30,9 @@
                         "courtOrderedLiquidation"
                     ]
                 },
+                "details": {
+                    "type": "string"
+                },
```

Comment:
can we add min/max to this like in court order details.  that way we won't run into any issues with the length exceeding the db field of size 2000.

https://github.com/bcgov/business-schemas/blob/278bf5e4d45035277f802879bea1c14fcd3792a2/src/registry_schemas/schemas/court_order.json#L37-L38